docs/index.rst

```diff
@@ -0,0 +1,27 @@
+.. google-auth documentation master file, created by
+   sphinx-quickstart on Thu Sep 22 12:50:15 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
```

Comment:
Removed.

Not sure what your second comment means, but I plan to have separate user's guide and reference similar to what I did for [urllib3](https://urllib3.readthedocs.io/en/latest/).